src/org/deepsymmetry/bytefield/core.cljs

```diff
@@ -531,57 +531,85 @@
   the standard `:box-above` style. You can change that by passing
   different attributes under the `:box-above-style` key (for example,
   use `{:box-above-style :box-above-related}` if the gap relates to
-  the preceding box)."
+  the preceding box).
+  
+  The background can be filled with a color passed with `:fill`."
   ([]
    (draw-gap nil nil))
   ([label]
    (draw-gap label nil))
   ([label attr-spec]
-   (let [{:keys [height gap edge gap-style box-above-style min-label-columns]
+   (let [attrs (eval-attribute-spec attr-spec)]
+   (let [{:keys [height gap edge gap-style box-above-style min-label-columns fill]
```

Comment:
Ok, it runs beautifully, and now I am looking more closely. One of the nice things of using an editor-integrated linter like clj-kondo is it will immediately suggest combining these two let statements into one. But that’s not infrastructure I would expect someone to set up to dip into a project like this. If keeping the let statements separate, we’d want to at least indent the inner one.

Reply:
Thanks for the suggestion, addressed in 2b1dba